### DISCUSSION TITLE
Official Tweaks HUDless option?

### DISCUSSION THREAD

**Discussion by Tenome** [emoose/OutRun2006Tweaks#250] [Suggestions]
### Describe the feature you'd like to see added

I think I might be the only person using the cheat engine address you provided (thanks for that), and the other option is HUDLess textures, so... Would this be possible as an option from Tweaks? That'd be really nice.

**Comment by emoose** [comment]
Have you noticed any hud parts that don't work with the cheat engine addr? Seemed like some C2C code might ignore it but maybe I'm wrong about that.

Might be possible to add some KB binding to toggle it, maybe something like ` or ~.

**Comment by Tenome** [comment]
All HUD elements are completely removed when I play with it. I've done time trials, online, etc. I haven't noticed anything slipping by.

**Comment by Tenome** [comment]
Closing since 0.5.1 has this as a feature now.

**Comment by Tenome** [comment]
Actually, is this limited to keyboard only? Can controller inputs also be made as an option? Kinda awkward to reach up and hit the hotkey. In the meantime I'll just use steam to make a binding for it.

**Comment by Tenome** [comment]
> Have you noticed any hud parts that don't work with the cheat engine addr? Seemed like some C2C code might ignore it but maybe I'm wrong about that.
> 
> Might be possible to add some KB binding to toggle it, maybe something like ` or ~.

This is super minor, but I noticed that if you hide the HUD while slipstreaming, the slipstream graphic will stay on screen (doesn't seem to go away when leaving slipstream either). You can get rid of it by going out of slipstreaming and then toggling the HUD again.
